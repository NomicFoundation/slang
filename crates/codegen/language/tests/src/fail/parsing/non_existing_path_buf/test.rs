#![allow(unused_crate_dependencies)]

codegen_language_macros::compile!(Language(
    name = Foo,
    root_item = Bar1,
    leading_trivia = Sequence([]),
    trailing_trivia = Sequence([]),
    versions = ["1.0.0", "2.0.0", "3.0.0"],
    sections = [Section(
        title = "Section One",
        topics = [
            Topic(
                title = "Topic One",
                notes_file = "README.md", // should be correct
                items = [Struct(
                    name = Bar1,
                    fields = (field = Required(NonTerminal(Bar2)))
                )]
            ),
            Topic(
                title = "Topic Two",
                notes_file = "foo/bar/non-existing-file.md",
                items = [Struct(
                    name = Bar2,
                    fields = (field = Required(NonTerminal(Bar1)))
                )]
            )
        ]
    )]
));

fn main() {}
