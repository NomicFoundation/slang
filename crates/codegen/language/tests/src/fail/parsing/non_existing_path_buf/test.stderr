error: Path not found: $WORKSPACE/foo/bar/non-existing-file.md
  --> src/fail/parsing/non_existing_path_buf/test.rs:22:30
   |
22 |                 notes_file = "foo/bar/non-existing-file.md",
   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
